python/cugraph/tests/test_betweenness_centrality.py

```diff
@@ -299,80 +298,96 @@ def prepare_test():
     gc.collect()
 
 
+# =============================================================================
+# Pytest Fixtures
+# =============================================================================
+DIRECTED = [pytest.param(d) for d in DIRECTED_GRAPH_OPTIONS]
+DATASETS_SMALL = [pytest.param(d) for d in utils.DATASETS_SMALL]
+DATASETS_UNRENUMBERED = [pytest.param(d) for d in utils.DATASETS_UNRENUMBERED]
+WEIGHTED_GRAPH_OPTIONS = [pytest.param(w) for w in WEIGHTED_GRAPH_OPTIONS]
+
+
+fixture_params_dts_sml = utils.genFixtureParamsProduct(
+    (DATASETS_SMALL, "grph"),
+    (DIRECTED, "dirctd"),
+    (WEIGHTED_GRAPH_OPTIONS, "wgtd_gph_opts"))
+
+fixture_params_dts_urnbrd = utils.genFixtureParamsProduct(
+    (DATASETS_UNRENUMBERED, "grph"),
+    (DIRECTED, "dirctd"),
+    (WEIGHTED_GRAPH_OPTIONS, "wgtd_gph_opts"))
+
+
+@pytest.fixture(scope="module", params=fixture_params_dts_sml)
+def Fixture_params_dts_sml(request):
```

Comment:
It might not be obvious that there's a variable for the fixture params and the fixture itself named almost the same.  I've noticed that other fixtures that come with pytest and elsewhere don't use a `Fixture_` prefix, so maybe change this to just `dts_small`, or maybe `dts_small_fixture` to differentiate from the variable of almost the same name?

Also, I found some of the abbreviations to not be as readable as just spelling out the words, like `datasets_small_fixture` or something similar.

Reply:
I renamed what was passed in `params` `small_graph_fixture_params` to stay consistent with the naming from other tests like `test_bfs`.